### DISCUSSION TITLE
Auth0

### DISCUSSION THREAD

**Discussion by evansims** [HassanZahirnia/laravel-package-ocean#7] [Package Suggestions]
Auth0's Laravel authentication and endpoint authorization SDK; [auth0/login](https://packagist.org/packages/auth0/login), https://github.com/auth0/laravel-auth0

**Comment by HassanZahirnia** [comment]
Hey Evan 👋
I will have that package added and push it to live site soon~~

Thanks for your work! 😊🙌

**Reply by evansims** [reply]
Thanks for creating Ocean, it's a great resource!

**Reply by HassanZahirnia** [reply]
Thanks for your kind words 🌸 your package is live on site now~
